src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -26,15 +26,18 @@
 class Range extends Constraint
 {
     const INVALID_CHARACTERS_ERROR = 'ad9a9798-7a99-4df7-8ce9-46e416a1e60b';
+    const NOT_IN_RANGE_ERROR = '04b91c99-a946-4221-afc5-e65ebac401eb';
     const TOO_HIGH_ERROR = '2d28afcb-e32e-45fb-a815-01c431a86a69';
     const TOO_LOW_ERROR = '76454e69-502c-46c5-9643-f447d837c4d5';
 
     protected static $errorNames = [
         self::INVALID_CHARACTERS_ERROR => 'INVALID_CHARACTERS_ERROR',
+        self::NOT_IN_RANGE_ERROR => 'NOT_IN_RANGE_ERROR',
         self::TOO_HIGH_ERROR => 'TOO_HIGH_ERROR',
         self::TOO_LOW_ERROR => 'TOO_LOW_ERROR',
     ];
 
+    public $notInRangeMessage = 'This value should be between {{ min }} and {{ max }}.';
```

Comment:
This new message should be added to the translation files (at least, the `en` one).

Reply:
Done 👍